packages/api-form-builder/src/plugins/graphql/form.ts

```diff
@@ -553,6 +554,22 @@ const plugin: GraphQLSchemaPlugin<FormBuilderContext> = {
                             }
                         }
 
+                        /**
+                         * Add meta fields.
+                         */
+                        for (let i = 0; i < submissions.length; i++) {
+                            const flattenExportMeta = flattenObj(
```

Comment:
Since this is really related to form submission meta key, let's maybe rename this to `flattenSubmissonMeta`. Having it more explicit gives the info that its logic is specific to form submission meta, which it is.

Let's do the same on the frontend side.